perception/multi_object_tracker/src/data_association/data_association.cpp

```diff
@@ -207,6 +207,13 @@ Eigen::MatrixXd DataAssociation::calcScoreMatrix(
         if (passed_gate) {
           const double min_iou = min_iou_matrix_(tracker_label, measurement_label);
           const double min_union_iou_area = 1e-2;
+          if (
+            measurement_object.shape.type == autoware_auto_perception_msgs::msg::Shape::POLYGON &&
+            area == 0 && measurement_object.shape.footprint.points.size() == 1) {
```

Comment:
```suggestion
            area == 0 && measurement_object.shape.footprint.points.size() == 1) {
```
Same as here https://github.com/autowarefoundation/autoware.universe/pull/6618/files#r1525002210